### DISCUSSION TITLE
reportUnusedDisableDirectives: Is the default 'off"?

### DISCUSSION THREAD

**Discussion by erosman** [eslint/eslint#18224] [General] [answered]
When testing, it appears that the default is "warn". 
(eslint@9.0.0-rc.0 & @stylistic/eslint-plugin-js@1.0.1)

> [Configuration Files (New)](https://eslint.org/docs/latest/use/configure/configuration-files-new)
> 
> reportUnusedDisableDirectives - A severity string indicating if and how unused disable and enable directives should be tracked and reported. For legacy compatibility, true is equivalent to "warn" and false is equivalent to "off". (default: "off")

See also:

https://github.com/eslint/eslint/blob/b185eb97ec60319cc39023e8615959dd598919ae/lib/config/default-config.js#L47

**Comment by mdjermanovic** [comment, marked as answer]
In ESLint v8, the default is "off". In ESLint v9, the default is "warn".

You can find the docs for prereleases (currently v9.0.0-rc.0) under https://eslint.org/docs/next/, or switch the version in the version selector:

![image](https://github.com/eslint/eslint/assets/44349756/86b7ae68-b0b7-4970-8695-7c4b1511b5f5)

https://eslint.org/docs/next/use/configure/configuration-files#configuration-objects

> `reportUnusedDisableDirectives` - A severity string indicating if and how unused disable and enable directives should be tracked and reported. For legacy compatibility, `true` is equivalent to `"warn"` and `false` is equivalent to `"off"`. (default: `"warn"`)
